api/src/index.ts

```diff
@@ -11,15 +14,23 @@ import { KibblesService } from "./services/kibbles";
 import { KittyItemsService } from "./services/kitty-items";
 import { MarketService } from "./services/market";
 import { SaleOfferHandler } from "./workers/sale-offer-handler";
+import { Model } from "objection";
 
 let knexInstance: Knex;
+const argv = yargs(hideBin(process.argv)).argv;
 
 async function run() {
   const config = getConfig();
 
   knexInstance = Knex({
     client: "postgresql",
-    connection: config.databaseUrl,
+    connection: {
+      connectionString: config.databaseUrl,
+      ssl:
+        process.env.NODE_ENV === "production"
+          ? { rejectUnauthorized: false }
+          : false,
```

Comment:
What's this for?

Reply:
Herokuy requires SSL connection to Postgres in production. In dev it's not necessary. This is the client config for enabling SSL connection.

Reply:
I think on dev you might be able to disable ssl directly through the connection string: `postgres://localhost/dbname?sslmode=disable`

Reply:
I played around with query strings as well. You have to *enable* it for Heroku using the client config (for some reason it's not the default, and Heroku wont let you edit the connection url).